elements/pfe-cta/src/pfe-cta.scss

```
@@ -21,7 +21,8 @@ $pfe-cta--Color--fallback: #003366;
   // hover variables
   --pfe-cta--BackgroundColor--hover:  transparent;
   --pfe-cta--BorderColor--hover: transparent;
-  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
+  --pfe-cta--Color--hover:  pfe-color(ui-link--hover);
```

Keep the interpolation:

```suggestion
  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
```